tests/testthat/helper-funs.R

```diff
@@ -0,0 +1,35 @@
+basicScatP <- function() {
+  ggplot2::ggplot(data = mtcars) +
+    ggplot2::geom_point(ggplot2::aes(x = wt, y = drat)) 
+}
+
+ggMarg2 <- function(type, ...) 
+  ggMarginal(p = basicScatP(), type = type, ...)
+
+funList <- 
+  list(
```

Comment:
ggplot subtitles just came to mind - I think they were introduced in 2.0? But this is fine for now. If ggplot2 does end up adding multiple new features that this becomes an issue we can address it then